src/vxingest/grib2_to_cb/grib_builder_parent.py

```diff
@@ -832,9 +833,9 @@ def build_document(self, queue_element):
             document_map[data_file_doc["id"]] = data_file_doc
             self.delete_idx_file(queue_element)
             return document_map
-        except FileNotFoundError:
+        except (FileNotFoundError, cfgrib.IOProblemError):
             logger.error(
-                "%s: Exception with builder build_document: file_name: %s, error: file not found - skipping this file",
+                "%s: Exception with builder build_document: file_name: %s, error: file not found or problem reading file - skipping this file",
```

Comment:
Ah, nice! So it was `cfgrib` that imports `gribapi.errors`. I like this `cfgrib.IOProblemError` signature you're using much better than `gribapi.errors.IOProblemError` - it's easier to figure out where the error's from.

Reply:
right. I identified the error from the debugger as I stepped through the code. I could have traced it down a little further.